client/lib/protect-form/index.tsx

```diff
@@ -1,10 +1,6 @@
-/** @format */
```

Comment:
did we intend to remove the formatting pragma?

Reply:
Yes, we got rid of the requirement and are auto-formatting nearly everything. I generally remove them when I thouch a file.